products/coder.md

```diff
@@ -0,0 +1,80 @@
+title: Coder
+category: server-app
+tags: remote cde terraform ide development docker kubernetes
+iconSlug: coder
+permalink: /coder
+versionCommand: coder version
+releasePolicyLink: https://coder.com/docs/install/releases#release-schedule
+changelogTemplate: https://github.com/coder/coder/releases/tag/v__LATEST__
+releaseDateColumn: true
+eoasColumn: Mainline
+eoasWarnThreshold: 30
+eolColumn: Stable
+eolWarnThreshold: 60
+eoesColumn: Security Support
+eoasWarnThreshold: 150
+
+auto:
+  methods:
+  - git: https://github.com/coder/coder.git
+
+releases:
+- releaseCycle: "2.13"
+  releaseDate: 2024-07-02
+  eoas: 2024-08-06
+  eol: 2024-09-03
+  eoes: false
+  latest: "2.13.1"
+  latestReleaseDate: 2024-07-20
+
+- releaseCycle: "2.12"
+  releaseDate: 2024-06-04
+  eoas: 2024-07-02
+  eol: 2024-08-06
+  eoes: false
+  latest: "2.12.5"
+  latestReleaseDate: 2024-07-22
+
+- releaseCycle: "2.11"
+  releaseDate: 2024-05-07
+  eoas: 2024-06-04
+  eol: 2024-07-02
+  eoes: false
+  latest: "2.11.3"
+  latestReleaseDate: 2024-06-21
+
+- releaseCycle: "2.10"
+  releaseDate: 2024-04-03
+  eoas: 2024-05-07
+  eol: 2024-06-04
+  eoes: false
+  latest: "2.10.3"
+  latestReleaseDate: 2024-05-22
+
+- releaseCycle: "2.9"
+  releaseDate: 2024-03-07
+  eoas: 2024-04-03
+  eol: 2024-05-07
+  eoes: true
+  latest: "2.9.4"
+  latestReleaseDate: 2024-04-22
+
+- releaseCycle: "2.8"
+  releaseDate: 2024-02-06
+  eoas: 2024-03-07
+  eol: 2024-04-03
+  eoes: true
+  latest: "2.8.4"
+  latestReleaseDate: 2024-03-09
+
+- releaseCycle: "2.7"
+  releaseDate: 2024-01-01
+  eoas: 2024-02-06
+  eol: 2024-03-07
+  eoes: true
+  latest: "2.7.3"
+  latestReleaseDate: 2024-03-04
+
+---
+
+> [Coder](https://github.com/coder/coder) is an open-source cloud development environment (CDE) that you host in your cloud or on-premises. With Coder, you can deploy environments that provide the infrastructure, IDEs, and tools your developers need. Upgrade to Coder Enterprise to gain enhanced security, governance, and observability for your platform teams.
```

Comment:
The product needs a description of the release and support policy. From what I understood:

- the last three versions are supported
- there are three stages of support:
  - mainline: bug and security fixes
  - stable: major bug and security fixes
  - security: only security fixes

Unfortunately endoflife.date structure is not suited for this kind of support. You used the `eoesColumn` for documenting security support but this does not fit : extended support should be reserved for support past EOL, such as commercial support. So given the security phase is a standard support phase, it should be documented in the `eolColumn`.

I propose to document both mainline and stable versions in the active support. I agree it's not ideal, but it's close to reality and best fit with endoflife.date way of documenting versions.

WDYT ?

Reply:
And given what I understood of the support policy, we could also apply the following rule:

- eoas(x) = releaseDate(x+2)
- eol(x) = releaseDate(x+3)

Reply:
This sounds nice. I would appreciate some help on how can we apply these rules?

Reply:
> Unfortunately endoflife.date structure is not suited for this kind of support. You used the eoesColumn for documenting security support but this does not fit : extended support should be reserved for support past EOL, such as commercial support. So given the security phase is a standard support phase, it should be documented in the eolColumn.

I agree.

Reply:
> I propose to document both mainline and stable versions in the active support. I agree it's not ideal, but it's close to reality and best fit with endoflife.date way of documenting versions.

I agree and we do support both of them. Its just that once a release is cut it does not get new features but only bug and security fixes.

Reply:
I am still unsure on how to document the previous 2 release from the current stable (in this case `2.11.x` and `2.10.x`) that will only get critical security fixes.

Reply:
I think I have got it. Please see the new changes. How can I change or preferably split it in 2 tables saying `Security and Bug Fixes Support` and `Security Support`?

Reply:
I think I have fixed it for now. And I like the current preview. I hope it will be automated.

Reply:
Unfortunately for now EOAS / EOL cannot be automated unless there is a page on the official website documenting the dates.